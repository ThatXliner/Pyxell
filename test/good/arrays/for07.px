
a = [1, 2, 3]
b = ['C', 'A']
for x, y in a step 2, b step -1 do
    print x, y
for t in a step -2, b step 2 do
    print t
