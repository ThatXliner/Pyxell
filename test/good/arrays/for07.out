1 A
3 C
3 C
